### ISSUE TITLE
err

### ISSUE THREAD

**Issue by antondavletov** [avocode/avocode#1799]
---
* **Version**: 2.17.2-web.1
* **Web/Desktop**: Web
* **Platform**: windows 10.0
* **Browser**: chrome 63.0.3239.84
* **Avocode UID**: 238058
---
```
TypeError: n.indexOf is not a function
    at https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:101:1378221
    at https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:1:53143
    at Be.__iterate (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:1:39275)
    at Be.findEntry (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:1:53112)
    at Be.find (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:1:51732)
    at e.value (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:101:1378191)
    at e.value (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:101:1376810)
    at e.value (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:101:694498)
    at r (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:17:397666)
    at p (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-web.min.js:17:397952)
    at h (https://cdn.app.avocode.com/dist/v-30ef9b2/js/avocode-
```

**Comment by vojtatranta** [comment]
Hi @antondavletov,
we know about this bug and we will fix today in the next release on the web.

Thank you!

**Comment by vojtatranta** [comment]
@antondavletov it's fixed in `2.17.2-web.2` currently deployed.
